.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,11 @@
 Please fill in this template.
 
 - [ ] Use a meaningful title for the pull request. Include the name of the package modified.
+- [ ] Is this just a bug fix, new functionality, or breaking change?
 - [ ] Have read [How To Contribute](https://github.com/Turfjs/turf/blob/master/CONTRIBUTING.md#how-to-contribute).
 - [ ] Run `npm test` at the sub modules where changes have occurred.
```

Comment:
Should they be running test across the whole repo? The change may work in the sub module, but have a flow on effect in another package.

Reply:
It is safer to run them at the root, but I think we also have more trustworthy CI now that should hopefully catch anything that slips through.

Reply:
I've changed it to root, but made it clear package level is available for active development.